docs/source/usage/01_usage.md

```diff
@@ -38,12 +38,16 @@ are shared among multiple roles:
 - `cifmw_architecture_repo`: (String) Path of the architecture repository on the controller node.
   Defaults to `~/src/github.com/openstack-k8s-operators/architecture`
 - `cifmw_architecture_scenario`: (String) The selected VA scenario to deploy.
-- `cifmw_ceph_target`: (String) the Ansible inventory group where ceph is deployed. Defaults to `computes`.
+- `cifmw_ceph_target`: (String) The Ansible inventory group where ceph is deployed. Defaults to `computes`.
```

Comment:
uh, those are unrelated changes, shouldn't they go in their own change? And why are they showing up just now? A change in the linter?

Reply:
I just fixed a few typos. The parameter description is the same, I just changed them so the first letter is uppercase to keep the docs page consistent